constants.py

```diff
@@ -63,12 +64,13 @@ def remove_comments(text):
 class Constants(dict): # type: ignore[type-arg]
     """Transforms dict to object, attributes can be accessed by dot notation."""
 
-    def __setattr__(self, name, value):
-        # type: (Text, Any) -> None
+    # Here `value` has the type Any because it parses and stores the values of
+    # contants defined in constants.ts file and we cannot define a single type
+    # which works for all of them.
+    def __setattr__(self, name: str, value: str) -> None:
```

Comment:
Type of `value` should be `Any` here.

Reply:
Thanks for pointing out. Fixed now. I guess mypy wasn't able to detect this because this process happens dynamically, and mypy is only static type checker.